web/src/app/services/data-store/data-store.service.ts

```diff
@@ -298,7 +298,7 @@ export class DataStoreService {
 
   lois$({id}: Survey): Observable<List<LocationOfInterest>> {
     return this.db
-      .collection(`${SURVEYS_COLLECTION_NAME}/${id}/lois`)
+      .collection(`${SURVEYS_COLLECTION_NAME}/${id}/lois`, ref => ref.orderBy('properties.name'))
```

Comment:
Thanks for the fix! The label shown in the UIs is a composite of name and ID, so you may need to sort this somewhere further down the stack. `loi.service.ts` has `getDisplayName()` which can be used to form the name for sorting. You can sort them in the lois exposed from that service, or further down in the UI layer if you think it's appropriate.

Reply:
I moved the sorting into the `loi.service.ts` file in the `getLocationsOfInterest$ `method. Sorry it took me so long to get around to it. I had a lot going on for a bit there, but I should be free to help out more often. So feel free to assign me some more issues. Also, this is my first time using RxJS, so let me know if you think there's a better place to put this than where it's at.